docs/book/v4/strategies/backed-enum.md

```diff
@@ -0,0 +1,101 @@
+# BackedEnum
+
+> Available since 4.8.0
+
+> Requires PHP 8.1+
+
+The `BackedEnumStrategy` provides **bidirectional conversion between strings 
+or integers and [Backed Enums](https://www.php.net/manual/en/language.enumerations.backed.php)**.
+
+The code examples below will use the following backed enum, representing a 
+genre of music:
+
+```php
+enum Genre: string
+{
+    case Pop   = 'pop';
+    case Blues = 'blues';
+    case Jazz  = 'jazz';
+}
+```
+
+## Basic usage
```

Comment:
For headlines the Chicago Manual of Style capitalization rules must be used:

Capitalize the first and the last word.
Capitalize nouns, pronouns, adjectives, verbs, adverbs, and subordinate conjunctions.
Lowercase articles (a, an, the), coordinating conjunctions, and prepositions.
Lowercase the ‘to’ in an infinitive (I want to play guitar).
The online tool [capitalizemytitle.com](https://capitalizemytitle.com/style/Chicago/) can help to capitalize and case convert the headlines.

```suggestion
## Basic Usage
```